conda/resolve.py

```diff
@@ -640,6 +640,7 @@ def generate_version_metrics(self, C, specs, include0=False):
         eqc = {}  # a C.minimize() objective: Dict[varname, coeff]
         eqv = {}  # a C.minimize() objective: Dict[varname, coeff]
         eqb = {}  # a C.minimize() objective: Dict[varname, coeff]
+        eqt = {}  # a C.minimize() objective: Dict[varname, coeff]
         sdict = {}  # Dict[package_name, Dist]
```

Comment:
Can we get some comments about what each of `eqc`, `eqv`, `eqb`, and `eqt` actually represent? @mcg1969 maybe if you add something briefly, either @msarahan or I can add additional detail helpful for people reading the code for the first time.

Reply:
Sure, that seems reasonable---if you want to do  it quickly, these are dictionaries containing weights for channel, version, build, and timestamp priority, respectively.